packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -166,7 +185,9 @@ export class Copy {
             fs.rmSync(downloadDir, {recursive: true});
             return {
                 success:true,
-                commandResponse: ZosFilesMessages.datasetCopiedSuccessfully.message
+                commandResponse: newDataSet
+                    ? ZosFilesMessages.newDataSetCreated.message + ` - "${toPds}"`
```

Comment:
We use the `ZosFilesMessages` class to define message strings in a common place so that they can be easily translated (in the future because we don't support i18n yet).

Please define the message with a `%s` placeholder so that the entire message is translatable. Then you can use the `util.format` function to replace the placeholder with a real value.
```typescript
    newDataSetCreated: {
        message: `New data set created - "%s"`
    },
```
```suggestion
                    ? util.format(ZosFilesMessages.newDataSetCreated.message, toPds)
```